src/lib/eslint/JavascriptEslintStrategy.ts

```diff
@@ -7,17 +7,23 @@ import path = require('path');
 import {ESLint} from 'eslint';
 
 const ES_CONFIG: ESLint.Options = {
-	"baseConfig": {},
```

Comment:
Do we not need the baseConfig() anymore?

Reply:
I didn't mean to get rid of that line, thanks for catching that.